documentation/images/README

```diff
@@ -1 +0,0 @@
-Images will be added here in the future.
```

Comment:
I cannot commant on the picture, so I will comment here. The picture might be a good start. Bu we need a new one without the Kubernetes part basically. Just:
* Kafka brokers
* Bridge
* one set of clients